inc/wpseo-functions.php

```diff
@@ -259,3 +259,50 @@ function wpseo_get_capabilities() {
 	}
 	return WPSEO_Capability_Manager_Factory::get()->get_capabilities();
 }
+
+
+/**
+ * Callback function to pass to the oEmbed's response data that will enable
+ * support for using the image and title set by the WordPress SEO plugin's fields. This
+ * address the concern where some social channels/subscribed use oEmebed data over OpenGraph data
+ * if both are present.
+ * 
+ * @param {array}  $data   - The oEmbed data
```

Comment:
Please remove the `{` and `}` and let the comment end with a full stop. This should be done for all the `@param` and `@return` docs

Reply:
Done, thanks!